source/com.microsoft.tfs.client.common.ui/src/com/microsoft/tfs/client/common/ui/messages.properties

```diff
@@ -1145,6 +1145,9 @@ NotesCheckinControl.NoteFieldNotConfiguredWithName=Checkin note field control is
 NotesCheckinControl.RequiredMessage=<Required>
 NotesCheckinControl.Title=Checkin Notes
 NullWorkItemControl.LabelTextFormat=WIControl ({0})
+OAuth2DeviceFlowCallbackDialog.DeviceLoginUrlText=To complete the authentication process, visist the following URL:
+OAuth2DeviceFlowCallbackDialog.DeviceUserCodeText=When prompted, enter the following code:
+OAuth2DeviceFlowCallbackDialog.SelectOkText=Once authenticated from the browser, select "OK".
```

Comment:
Not explicitly, how do you run MessageTest?  I ran the cmd line build and I was hoping the build would run all tests for me.